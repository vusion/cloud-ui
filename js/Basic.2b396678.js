(window["webpackJsonp"]=window["webpackJsonp"]||[]).push([["Basic"],{"02bc":function(t,a,e){"use strict";e.r(a);var s=function(){var t=this,a=t.$createElement,e=t._self._c||a;return e("u-article",[e("h1",{attrs:{id:"utext-文本"}},[t._v("UText 文本 "),e("u-link",{staticClass:"heading-anchor",attrs:{to:{hash:"#utext-文本"}}},[t._v("#")])],1),e("u-toc",[e("u-toc-item",{attrs:{label:"基础示例",to:"examples"}},[e("u-toc-item",{attrs:{label:"设置尺寸",to:{path:"examples",hash:"#设置尺寸"}}}),e("u-toc-item",{attrs:{label:"设置颜色",to:{path:"examples",hash:"#设置颜色"}}}),e("u-toc-item",{attrs:{label:"块级展示",to:{path:"examples",hash:"#块级展示"}}}),e("u-toc-item",{attrs:{label:"文本过长",to:{path:"examples",hash:"#文本过长"}}})],1),"development"===t.NODE_ENV?e("u-toc-item",{attrs:{label:"内置区块",to:"blocks"}},[e("u-toc-item",{attrs:{label:"普通文本",to:{path:"blocks",hash:"#普通文本"}}}),e("u-toc-item",{attrs:{label:"小号文本",to:{path:"blocks",hash:"#小号文本"}}}),e("u-toc-item",{attrs:{label:"大号文本",to:{path:"blocks",hash:"#大号文本"}}}),e("u-toc-item",{attrs:{label:"主要文本",to:{path:"blocks",hash:"#主要文本"}}}),e("u-toc-item",{attrs:{label:"次要文本",to:{path:"blocks",hash:"#次要文本"}}}),e("u-toc-item",{attrs:{label:"成功文本",to:{path:"blocks",hash:"#成功文本"}}}),e("u-toc-item",{attrs:{label:"警告文本",to:{path:"blocks",hash:"#警告文本"}}}),e("u-toc-item",{attrs:{label:"错误文本",to:{path:"blocks",hash:"#错误文本"}}}),e("u-toc-item",{attrs:{label:"禁默文本",to:{path:"blocks",hash:"#禁默文本"}}}),e("u-toc-item",{attrs:{label:"文本过长省略",to:{path:"blocks",hash:"#文本过长省略"}}})],1):t._e(),e("u-toc-item",{attrs:{label:"API",to:{path:"api",hash:""}}},[e("u-toc-item",{attrs:{label:"Props/Attrs",to:{path:"api",hash:"#propsattrs"}}}),e("u-toc-item",{attrs:{label:"Slots",to:{path:"api",hash:"#slots"}}}),e("u-toc-item",{attrs:{label:"Events",to:{path:"api",hash:"#events"}}})],1)],1),e("p",[e("s-component-labels",{attrs:{labels:["行内展示"]}})],1),e("p",[t._v("用于在业务中快速调整文本尺寸、颜色等特性。")]),e("u-h2-tabs",{attrs:{router:""}},[e("u-h2-tab",{attrs:{title:"基础示例",to:"examples"}}),"development"===t.NODE_ENV?e("u-h2-tab",{attrs:{title:"内置区块",to:"blocks"}}):t._e(),e("u-h2-tab",{attrs:{title:"API",to:"api"}})],1),e("router-view")],1)},l=[],o=e("2877"),h={},r=Object(o["a"])(h,s,l,!1,null,null,null);a["default"]=r.exports},2222:function(t,a,e){"use strict";e.r(a);var s=function(){var t=this,a=t.$createElement,e=t._self._c||a;return e("u-article",[e("h1",{attrs:{id:"uavatar-头像"}},[t._v("UAvatar 头像 "),e("u-link",{staticClass:"heading-anchor",attrs:{to:{hash:"#uavatar-头像"}}},[t._v("#")])],1),e("u-toc",[e("u-toc-item",{attrs:{label:"基础示例",to:"examples"}},[e("u-toc-item",{attrs:{label:"基本用法",to:{path:"examples",hash:"#基本用法"}}}),e("u-toc-item",{attrs:{label:"大小扩展",to:{path:"examples",hash:"#大小扩展"}}}),e("u-toc-item",{attrs:{label:"自定义图片",to:{path:"examples",hash:"#自定义图片"}}}),e("u-toc-item",{attrs:{label:"配合 UBadge 使用",to:{path:"examples",hash:"#配合-ubadge-使用"}}}),e("u-toc-item",{attrs:{label:"配合 UNavbar 使用",to:{path:"examples",hash:"#配合-unavbar-使用"}}})],1),"development"===t.NODE_ENV?e("u-toc-item",{attrs:{label:"内置区块",to:"blocks"}},[e("u-toc-item",{attrs:{label:"默认头像",to:{path:"blocks",hash:"#默认头像"}}}),e("u-toc-item",{attrs:{label:"头像2",to:{path:"blocks",hash:"#头像2"}}}),e("u-toc-item",{attrs:{label:"头像3",to:{path:"blocks",hash:"#头像3"}}}),e("u-toc-item",{attrs:{label:"迷你头像",to:{path:"blocks",hash:"#迷你头像"}}}),e("u-toc-item",{attrs:{label:"大头像",to:{path:"blocks",hash:"#大头像"}}}),e("u-toc-item",{attrs:{label:"添加徽章",to:{path:"blocks",hash:"#添加徽章"}}})],1):t._e(),e("u-toc-item",{attrs:{label:"API",to:{path:"api",hash:""}}},[e("u-toc-item",{attrs:{label:"Props/Attrs",to:{path:"api",hash:"#propsattrs"}}}),e("u-toc-item",{attrs:{label:"Slots",to:{path:"api",hash:"#slots"}}})],1)],1),e("p",[e("s-component-labels",{attrs:{labels:["行内展示"]}})],1),e("p",[t._v("顶部导航栏或页面中局部使用的用户头像。")]),e("u-h2-tabs",{attrs:{router:""}},[e("u-h2-tab",{attrs:{title:"基础示例",to:"examples"}}),"development"===t.NODE_ENV?e("u-h2-tab",{attrs:{title:"内置区块",to:"blocks"}}):t._e(),e("u-h2-tab",{attrs:{title:"API",to:"api"}})],1),e("router-view")],1)},l=[],o=e("2877"),h={},r=Object(o["a"])(h,s,l,!1,null,null,null);a["default"]=r.exports},"2dce":function(t,a,e){"use strict";e.r(a);var s=function(){var t=this,a=t.$createElement,e=t._self._c||a;return e("u-article",[e("h1",{attrs:{id:"ucopy-复制"}},[t._v("UCopy 复制 "),e("u-link",{staticClass:"heading-anchor",attrs:{to:{hash:"#ucopy-复制"}}},[t._v("#")])],1),e("u-toc",[e("u-toc-item",{attrs:{label:"基础示例",to:"examples"}},[e("u-toc-item",{attrs:{label:"基本用法",to:{path:"examples",hash:"#基本用法"}}}),e("u-toc-item",{attrs:{label:"触发元素",to:{path:"examples",hash:"#触发元素"}}}),e("u-toc-item",{attrs:{label:"提示内容",to:{path:"examples",hash:"#提示内容"}}}),e("u-toc-item",{attrs:{label:"提示方式",to:{path:"examples",hash:"#提示方式"}}}),e("u-toc-item",{attrs:{label:"提示位置",to:{path:"examples",hash:"#提示位置"}}}),e("u-toc-item",{attrs:{label:"提示时长",to:{path:"examples",hash:"#提示时长"}}}),e("u-toc-item",{attrs:{label:"禁用状态",to:{path:"examples",hash:"#禁用状态"}}})],1),"development"===t.NODE_ENV?e("u-toc-item",{attrs:{label:"内置区块",to:"blocks"}},[e("u-toc-item",{attrs:{label:"复制链接",to:{path:"blocks",hash:"#复制链接"}}})],1):t._e(),e("u-toc-item",{attrs:{label:"API",to:{path:"api",hash:""}}},[e("u-toc-item",{attrs:{label:"Props/Attrs",to:{path:"api",hash:"#propsattrs"}}}),e("u-toc-item",{attrs:{label:"Slots",to:{path:"api",hash:"#slots"}}}),e("u-toc-item",{attrs:{label:"Events",to:{path:"api",hash:"#events"}}})],1)],1),e("p",[t._v("复制文本到剪切板。")]),e("u-h2-tabs",{attrs:{router:""}},[e("u-h2-tab",{attrs:{title:"基础示例",to:"examples"}}),"development"===t.NODE_ENV?e("u-h2-tab",{attrs:{title:"内置区块",to:"blocks"}}):t._e(),e("u-h2-tab",{attrs:{title:"API",to:"api"}})],1),e("router-view")],1)},l=[],o=e("2877"),h={},r=Object(o["a"])(h,s,l,!1,null,null,null);a["default"]=r.exports},"360e":function(t,a,e){"use strict";e.r(a);var s=function(){var t=this,a=t.$createElement,e=t._self._c||a;return e("u-article",[e("h1",{attrs:{id:"ubase-基础"}},[t._v("UBase 基础 "),e("u-link",{staticClass:"heading-anchor",attrs:{to:{hash:"#ubase-基础"}}},[t._v("#")])],1),e("u-toc",[e("u-toc-item",{attrs:{label:"基础示例",to:"examples"}},[e("u-toc-item",{attrs:{label:"标题",to:{path:"examples",hash:"#标题"}}}),e("u-toc-item",{attrs:{label:"链接",to:{path:"examples",hash:"#链接"}}}),e("u-toc-item",{attrs:{label:"段落",to:{path:"examples",hash:"#段落"}}}),e("u-toc-item",{attrs:{label:"水平线",to:{path:"examples",hash:"#水平线"}}}),e("u-toc-item",{attrs:{label:"特殊文字",to:{path:"examples",hash:"#特殊文字"}}}),e("u-toc-item",{attrs:{label:"列表",to:{path:"examples",hash:"#列表"}}}),e("u-toc-item",{attrs:{label:"描述列表",to:{path:"examples",hash:"#描述列表"}}}),e("u-toc-item",{attrs:{label:"引用块",to:{path:"examples",hash:"#引用块"}}}),e("u-toc-item",{attrs:{label:"代码块",to:{path:"examples",hash:"#代码块"}}}),e("u-toc-item",{attrs:{label:"按键",to:{path:"examples",hash:"#按键"}}})],1),"development"===t.NODE_ENV?e("u-toc-item",{attrs:{label:"内置区块",to:"blocks"}},[e("u-toc-item",{attrs:{label:"一级标题",to:{path:"blocks",hash:"#一级标题"}}}),e("u-toc-item",{attrs:{label:"二级标题",to:{path:"blocks",hash:"#二级标题"}}}),e("u-toc-item",{attrs:{label:"三级标题",to:{path:"blocks",hash:"#三级标题"}}}),e("u-toc-item",{attrs:{label:"四级标题",to:{path:"blocks",hash:"#四级标题"}}}),e("u-toc-item",{attrs:{label:"段落",to:{path:"blocks",hash:"#段落"}}}),e("u-toc-item",{attrs:{label:"水平线",to:{path:"blocks",hash:"#水平线"}}}),e("u-toc-item",{attrs:{label:"引用块",to:{path:"blocks",hash:"#引用块"}}})],1):t._e(),e("u-toc-item",{attrs:{label:"API",to:{path:"api",hash:""}}})],1),e("p",[t._v("Cloud UI 在 Normalize.css 的基础上，对元素做了基础的排版。")]),e("u-h2-tabs",{attrs:{router:""}},[e("u-h2-tab",{attrs:{title:"基础示例",to:"examples"}}),"development"===t.NODE_ENV?e("u-h2-tab",{attrs:{title:"内置区块",to:"blocks"}}):t._e(),e("u-h2-tab",{attrs:{title:"API",to:"api"}})],1),e("router-view")],1)},l=[],o=e("2877"),h={},r=Object(o["a"])(h,s,l,!1,null,null,null);a["default"]=r.exports},"4d83":function(t,a,e){"use strict";e.r(a);var s=function(){var t=this,a=t.$createElement,e=t._self._c||a;return e("u-article",[e("h1",{attrs:{id:"uactions-操作项"}},[t._v("UActions 操作项 "),e("u-link",{staticClass:"heading-anchor",attrs:{to:{hash:"#uactions-操作项"}}},[t._v("#")])],1),e("u-toc",[e("u-toc-item",{attrs:{label:"基础示例",to:"examples"}},[e("u-toc-item",{attrs:{label:"基本用法",to:{path:"examples",hash:"#基本用法"}}}),e("u-toc-item",{attrs:{label:"使用指令",to:{path:"examples",hash:"#使用指令"}}})],1),e("u-toc-item",{attrs:{label:"UActions API",to:{path:"api",hash:"#uactions-api"}}},[e("u-toc-item",{attrs:{label:"Props/Attrs",to:{path:"api",hash:"#propsattrs"}}}),e("u-toc-item",{attrs:{label:"Slots",to:{path:"api",hash:"#slots"}}})],1),e("u-toc-item",{attrs:{label:"UAction API",to:{path:"api",hash:"#uaction-api"}}},[e("u-toc-item",{attrs:{label:"Props/Attrs",to:{path:"api",hash:"#propsattrs-2"}}}),e("u-toc-item",{attrs:{label:"Events",to:{path:"api",hash:"#events"}}})],1)],1),e("u-h2-tabs",{attrs:{router:""}},[e("u-h2-tab",{attrs:{title:"基础示例",to:"examples"}}),e("u-h2-tab",{attrs:{title:"API",to:"api"}})],1),e("router-view")],1)},l=[],o=e("2877"),h={},r=Object(o["a"])(h,s,l,!1,null,null,null);a["default"]=r.exports},"7db8":function(t,a,e){"use strict";e.r(a);var s=function(){var t=this,a=t.$createElement,e=t._self._c||a;return e("u-article",[e("h1",{attrs:{id:"iicon-图标"}},[t._v("IIcon 图标 "),e("u-link",{staticClass:"heading-anchor",attrs:{to:{hash:"#iicon-图标"}}},[t._v("#")])],1),e("u-toc",[e("u-toc-item",{attrs:{label:"基础示例",to:"examples"}},[e("u-toc-item",{attrs:{label:"基本用法",to:{path:"examples",hash:"#基本用法"}}}),e("u-toc-item",{attrs:{label:"所有图标",to:{path:"examples",hash:"#所有图标"}}})],1),e("u-toc-item",{attrs:{label:"API",to:{path:"api",hash:""}}},[e("u-toc-item",{attrs:{label:"Props/Attrs",to:{path:"api",hash:"#propsattrs"}}})],1)],1),e("p",[e("s-component-labels",{attrs:{labels:["图标组件","行内展示"]}})],1),e("u-h2-tabs",{attrs:{router:""}},[e("u-h2-tab",{attrs:{title:"基础示例",to:"examples"}}),e("u-h2-tab",{attrs:{title:"API",to:"api"}})],1),e("router-view")],1)},l=[],o=e("2877"),h={},r=Object(o["a"])(h,s,l,!1,null,null,null);a["default"]=r.exports},"8af5":function(t,a,e){"use strict";e.r(a);var s=function(){var t=this,a=t.$createElement,e=t._self._c||a;return e("u-article",[e("h1",{attrs:{id:"ulabel-标签"}},[t._v("ULabel 标签 "),e("u-link",{staticClass:"heading-anchor",attrs:{to:{hash:"#ulabel-标签"}}},[t._v("#")])],1),e("u-toc",[e("u-toc-item",{attrs:{label:"基础示例",to:"examples"}},[e("u-toc-item",{attrs:{label:"基本用法",to:{path:"examples",hash:"#基本用法"}}}),e("u-toc-item",{attrs:{label:"可删除标签",to:{path:"examples",hash:"#可删除标签"}}}),e("u-toc-item",{attrs:{label:"设置尺寸",to:{path:"examples",hash:"#设置尺寸"}}}),e("u-toc-item",{attrs:{label:"块级展示",to:{path:"examples",hash:"#块级展示"}}})],1),"development"===t.NODE_ENV?e("u-toc-item",{attrs:{label:"内置区块",to:"blocks"}},[e("u-toc-item",{attrs:{label:"普通标签",to:{path:"blocks",hash:"#普通标签"}}}),e("u-toc-item",{attrs:{label:"主要标签",to:{path:"blocks",hash:"#主要标签"}}}),e("u-toc-item",{attrs:{label:"成功标签",to:{path:"blocks",hash:"#成功标签"}}}),e("u-toc-item",{attrs:{label:"警告标签",to:{path:"blocks",hash:"#警告标签"}}}),e("u-toc-item",{attrs:{label:"错误标签",to:{path:"blocks",hash:"#错误标签"}}}),e("u-toc-item",{attrs:{label:"可删除标签",to:{path:"blocks",hash:"#可删除标签"}}}),e("u-toc-item",{attrs:{label:"小标签",to:{path:"blocks",hash:"#小标签"}}}),e("u-toc-item",{attrs:{label:"大标签",to:{path:"blocks",hash:"#大标签"}}})],1):t._e(),e("u-toc-item",{attrs:{label:"API",to:{path:"api",hash:""}}},[e("u-toc-item",{attrs:{label:"Props/Attrs",to:{path:"api",hash:"#propsattrs"}}}),e("u-toc-item",{attrs:{label:"Slots",to:{path:"api",hash:"#slots"}}}),e("u-toc-item",{attrs:{label:"Events",to:{path:"api",hash:"#events"}}})],1)],1),e("p",[e("s-component-labels",{attrs:{labels:["行内展示"]}})],1),e("p",[t._v("对事物进行标记和分类。")]),e("u-h2-tabs",{attrs:{router:""}},[e("u-h2-tab",{attrs:{title:"基础示例",to:"examples"}}),"development"===t.NODE_ENV?e("u-h2-tab",{attrs:{title:"内置区块",to:"blocks"}}):t._e(),e("u-h2-tab",{attrs:{title:"API",to:"api"}})],1),e("router-view")],1)},l=[],o=e("2877"),h={},r=Object(o["a"])(h,s,l,!1,null,null,null);a["default"]=r.exports},b292:function(t,a,e){"use strict";e.r(a);var s=function(){var t=this,a=t.$createElement,e=t._self._c||a;return e("u-article",[e("h1",{attrs:{id:"ubadge-徽章"}},[t._v("UBadge 徽章 "),e("u-link",{staticClass:"heading-anchor",attrs:{to:{hash:"#ubadge-徽章"}}},[t._v("#")])],1),e("u-toc",[e("u-toc-item",{attrs:{label:"基础示例",to:"examples"}},[e("u-toc-item",{attrs:{label:"基本用法",to:{path:"examples",hash:"#基本用法"}}}),e("u-toc-item",{attrs:{label:"右上角",to:{path:"examples",hash:"#右上角"}}}),e("u-toc-item",{attrs:{label:"最大值",to:{path:"examples",hash:"#最大值"}}}),e("u-toc-item",{attrs:{label:"小圆点",to:{path:"examples",hash:"#小圆点"}}})],1),"development"===t.NODE_ENV?e("u-toc-item",{attrs:{label:"内置区块",to:"blocks"}},[e("u-toc-item",{attrs:{label:"文字徽章",to:{path:"blocks",hash:"#文字徽章"}}}),e("u-toc-item",{attrs:{label:"数字徽章",to:{path:"blocks",hash:"#数字徽章"}}}),e("u-toc-item",{attrs:{label:"右上角徽章",to:{path:"blocks",hash:"#右上角徽章"}}}),e("u-toc-item",{attrs:{label:"最大值",to:{path:"blocks",hash:"#最大值"}}}),e("u-toc-item",{attrs:{label:"小圆点",to:{path:"blocks",hash:"#小圆点"}}}),e("u-toc-item",{attrs:{label:"右上角小圆点",to:{path:"blocks",hash:"#右上角小圆点"}}})],1):t._e(),e("u-toc-item",{attrs:{label:"API",to:{path:"api",hash:""}}},[e("u-toc-item",{attrs:{label:"Props/Attrs",to:{path:"api",hash:"#propsattrs"}}}),e("u-toc-item",{attrs:{label:"Slots",to:{path:"api",hash:"#slots"}}})],1)],1),e("p",[e("s-component-labels",{attrs:{labels:["行内展示"]}})],1),e("p",[t._v("用于显示消息条数和提醒用户处理的角标。")]),e("u-h2-tabs",{attrs:{router:""}},[e("u-h2-tab",{attrs:{title:"基础示例",to:"examples"}}),"development"===t.NODE_ENV?e("u-h2-tab",{attrs:{title:"内置区块",to:"blocks"}}):t._e(),e("u-h2-tab",{attrs:{title:"API",to:"api"}})],1),e("router-view")],1)},l=[],o=e("2877"),h={},r=Object(o["a"])(h,s,l,!1,null,null,null);a["default"]=r.exports},b941:function(t,a,e){"use strict";e.r(a);var s=function(){var t=this,a=t.$createElement,e=t._self._c||a;return e("u-article",[e("h1",{attrs:{id:"ubutton-按钮"}},[t._v("UButton 按钮 "),e("u-link",{staticClass:"heading-anchor",attrs:{to:{hash:"#ubutton-按钮"}}},[t._v("#")])],1),e("u-toc",[e("u-toc-item",{attrs:{label:"基础示例",to:"examples"}},[e("u-toc-item",{attrs:{label:"基本用法",to:{path:"examples",hash:"#基本用法"}}}),e("u-toc-item",{attrs:{label:"设置形状",to:{path:"examples",hash:"#设置形状"}}}),e("u-toc-item",{attrs:{label:"禁用状态",to:{path:"examples",hash:"#禁用状态"}}}),e("u-toc-item",{attrs:{label:"设置尺寸",to:{path:"examples",hash:"#设置尺寸"}}}),e("u-toc-item",{attrs:{label:"块级展示",to:{path:"examples",hash:"#块级展示"}}}),e("u-toc-item",{attrs:{label:"图标",to:{path:"examples",hash:"#图标"}}}),e("u-toc-item",{attrs:{label:"加载中",to:{path:"examples",hash:"#加载中"}}}),e("u-toc-item",{attrs:{label:"链接",to:{path:"examples",hash:"#链接"}}})],1),"development"===t.NODE_ENV?e("u-toc-item",{attrs:{label:"内置区块",to:"blocks"}},[e("u-toc-item",{attrs:{label:"主要按钮",to:{path:"blocks",hash:"#主要按钮"}}}),e("u-toc-item",{attrs:{label:"次要按钮",to:{path:"blocks",hash:"#次要按钮"}}}),e("u-toc-item",{attrs:{label:"创建按钮",to:{path:"blocks",hash:"#创建按钮"}}}),e("u-toc-item",{attrs:{label:"刷新按钮",to:{path:"blocks",hash:"#刷新按钮"}}}),e("u-toc-item",{attrs:{label:"危险按钮",to:{path:"blocks",hash:"#危险按钮"}}}),e("u-toc-item",{attrs:{label:"圆角按钮",to:{path:"blocks",hash:"#圆角按钮"}}}),e("u-toc-item",{attrs:{label:"小按钮",to:{path:"blocks",hash:"#小按钮"}}}),e("u-toc-item",{attrs:{label:"迷你按钮",to:{path:"blocks",hash:"#迷你按钮"}}})],1):t._e(),e("u-toc-item",{attrs:{label:"API",to:{path:"api",hash:""}}},[e("u-toc-item",{attrs:{label:"Props/Attrs",to:{path:"api",hash:"#propsattrs"}}}),e("u-toc-item",{attrs:{label:"Slots",to:{path:"api",hash:"#slots"}}}),e("u-toc-item",{attrs:{label:"Events",to:{path:"api",hash:"#events"}}})],1)],1),e("p",[e("s-component-labels",{attrs:{labels:["路由链接","行内展示"]}})],1),e("p",[t._v("用于触发一个即时操作。")]),e("u-h2-tabs",{attrs:{router:""}},[e("u-h2-tab",{attrs:{title:"基础示例",to:"examples"}}),"development"===t.NODE_ENV?e("u-h2-tab",{attrs:{title:"内置区块",to:"blocks"}}):t._e(),e("u-h2-tab",{attrs:{title:"API",to:"api"}})],1),e("router-view")],1)},l=[],o=e("2877"),h={},r=Object(o["a"])(h,s,l,!1,null,null,null);a["default"]=r.exports},f46d:function(t,a,e){"use strict";e.r(a);var s=function(){var t=this,a=t.$createElement,e=t._self._c||a;return e("u-article",[e("h1",{attrs:{id:"ulink-链接"}},[t._v("ULink 链接 "),e("u-link",{staticClass:"heading-anchor",attrs:{to:{hash:"#ulink-链接"}}},[t._v("#")])],1),e("u-toc",[e("u-toc-item",{attrs:{label:"基础示例",to:"examples"}},[e("u-toc-item",{attrs:{label:"基本用法",to:{path:"examples",hash:"#基本用法"}}}),e("u-toc-item",{attrs:{label:"设置颜色",to:{path:"examples",hash:"#设置颜色"}}}),e("u-toc-item",{attrs:{label:"禁用状态",to:{path:"examples",hash:"#禁用状态"}}}),e("u-toc-item",{attrs:{label:"块级展示",to:{path:"examples",hash:"#块级展示"}}})],1),"development"===t.NODE_ENV?e("u-toc-item",{attrs:{label:"内置区块",to:"blocks"}},[e("u-toc-item",{attrs:{label:"普通链接",to:{path:"blocks",hash:"#普通链接"}}}),e("u-toc-item",{attrs:{label:"路由链接",to:{path:"blocks",hash:"#路由链接"}}}),e("u-toc-item",{attrs:{label:"成功链接",to:{path:"blocks",hash:"#成功链接"}}}),e("u-toc-item",{attrs:{label:"警告链接",to:{path:"blocks",hash:"#警告链接"}}}),e("u-toc-item",{attrs:{label:"危险链接",to:{path:"blocks",hash:"#危险链接"}}}),e("u-toc-item",{attrs:{label:"禁用链接",to:{path:"blocks",hash:"#禁用链接"}}})],1):t._e(),e("u-toc-item",{attrs:{label:"API",to:{path:"api",hash:""}}},[e("u-toc-item",{attrs:{label:"Props/Attrs",to:{path:"api",hash:"#propsattrs"}}}),e("u-toc-item",{attrs:{label:"Slots",to:{path:"api",hash:"#slots"}}}),e("u-toc-item",{attrs:{label:"Events",to:{path:"api",hash:"#events"}}})],1)],1),e("p",[e("s-component-labels",{attrs:{labels:["路由链接","行内展示"]}})],1),e("p",[t._v("文字超链接。")]),e("u-h2-tabs",{attrs:{router:""}},[e("u-h2-tab",{attrs:{title:"基础示例",to:"examples"}}),"development"===t.NODE_ENV?e("u-h2-tab",{attrs:{title:"内置区块",to:"blocks"}}):t._e(),e("u-h2-tab",{attrs:{title:"API",to:"api"}})],1),e("router-view")],1)},l=[],o=e("2877"),h={},r=Object(o["a"])(h,s,l,!1,null,null,null);a["default"]=r.exports}}]);