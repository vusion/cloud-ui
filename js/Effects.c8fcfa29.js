(window["webpackJsonp"]=window["webpackJsonp"]||[]).push([["Effects"],{"5e5a":function(t,a,e){"use strict";e.r(a);var s=function(){var t=this,a=t.$createElement,e=t._self._c||a;return e("u-article",[e("h1",{attrs:{id:"eripple-波纹"}},[t._v("ERipple 波纹 "),e("u-link",{staticClass:"heading-anchor",attrs:{to:{hash:"#eripple-波纹"}}},[t._v("#")])],1),e("u-toc",[e("u-toc-item",{attrs:{label:"基础示例",to:"examples"}},[e("u-toc-item",{attrs:{label:"基本用法",to:{path:"examples",hash:"#基本用法"}}})],1),e("u-toc-item",{attrs:{label:"API",to:{path:"api",hash:""}}},[e("u-toc-item",{attrs:{label:"Props/Attrs",to:{path:"api",hash:"#propsattrs"}}})],1)],1),e("p",[e("s-component-labels",{attrs:{labels:["特效组件"]}})],1),e("p",[t._v("波纹，插入到需要添加波纹的元素或组件内部")]),e("u-h2-tabs",{attrs:{router:""}},[e("u-h2-tab",{attrs:{title:"基础示例",to:"examples"}}),e("u-h2-tab",{attrs:{title:"API",to:"api"}})],1),e("router-view")],1)},l=[],r=e("2877"),o={},u=Object(r["a"])(o,s,l,!1,null,null,null);a["default"]=u.exports},c601:function(t,a,e){"use strict";e.r(a);var s=function(){var t=this,a=t.$createElement,e=t._self._c||a;return e("u-article",[e("h1",{attrs:{id:"ewatermark-水印"}},[t._v("EWatermark 水印 "),e("u-link",{staticClass:"heading-anchor",attrs:{to:{hash:"#ewatermark-水印"}}},[t._v("#")])],1),e("u-toc",[e("u-toc-item",{attrs:{label:"基础示例",to:"examples"}},[e("u-toc-item",{attrs:{label:"基本用法",to:{path:"examples",hash:"#基本用法"}}})],1),e("u-toc-item",{attrs:{label:"API",to:{path:"api",hash:""}}},[e("u-toc-item",{attrs:{label:"Props/Attrs",to:{path:"api",hash:"#propsattrs"}}})],1)],1),e("u-h2-tabs",{attrs:{router:""}},[e("u-h2-tab",{attrs:{title:"基础示例",to:"examples"}}),e("u-h2-tab",{attrs:{title:"API",to:"api"}})],1),e("router-view")],1)},l=[],r=e("2877"),o={},u=Object(r["a"])(o,s,l,!1,null,null,null);a["default"]=u.exports},cbe4:function(t,a,e){"use strict";e.r(a);var s=function(){var t=this,a=t.$createElement,e=t._self._c||a;return e("u-article",[e("h1",{attrs:{id:"ucarousel-幻灯片"}},[t._v("UCarousel 幻灯片 "),e("u-link",{staticClass:"heading-anchor",attrs:{to:{hash:"#ucarousel-幻灯片"}}},[t._v("#")])],1),e("u-toc",[e("u-toc-item",{attrs:{label:"基础示例",to:"examples"}},[e("u-toc-item",{attrs:{label:"基本用法",to:{path:"examples",hash:"#基本用法"}}}),e("u-toc-item",{attrs:{label:"动画",to:{path:"examples",hash:"#动画"}}}),e("u-toc-item",{attrs:{label:"自动播放与循环",to:{path:"examples",hash:"#自动播放与循环"}}})],1),e("u-toc-item",{attrs:{label:"UCarousel API",to:{path:"api",hash:"#ucarousel-api"}}},[e("u-toc-item",{attrs:{label:"Props/Attrs",to:{path:"api",hash:"#propsattrs"}}}),e("u-toc-item",{attrs:{label:"Slots",to:{path:"api",hash:"#slots"}}}),e("u-toc-item",{attrs:{label:"Events",to:{path:"api",hash:"#events"}}})],1),e("u-toc-item",{attrs:{label:"UCarouselItem API",to:{path:"api",hash:"#ucarouselitem-api"}}},[e("u-toc-item",{attrs:{label:"Props/Attrs",to:{path:"api",hash:"#propsattrs-2"}}}),e("u-toc-item",{attrs:{label:"Slots",to:{path:"api",hash:"#slots-2"}}})],1)],1),e("u-h2-tabs",{attrs:{router:""}},[e("u-h2-tab",{attrs:{title:"基础示例",to:"examples"}}),e("u-h2-tab",{attrs:{title:"API",to:"api"}})],1),e("router-view")],1)},l=[],r=e("2877"),o={},u=Object(r["a"])(o,s,l,!1,null,null,null);a["default"]=u.exports}}]);