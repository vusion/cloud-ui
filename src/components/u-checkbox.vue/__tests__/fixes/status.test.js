import Vue from 'vue';
import { UCheckbox } from '../../index.js';
import { mount } from '@vue/test-utils';

describe('u-checkbox.vue', () => {
    it('value 设置文本值渲染', () => {
        const wrapper = mount(UCheckbox, {
            propsData: {
                value: '23333',
            },
        });
        expect(wrapper).toMatchSnapshot();
    });
});
