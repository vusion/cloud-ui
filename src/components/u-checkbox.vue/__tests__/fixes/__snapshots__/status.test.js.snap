// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`u-checkbox.vue value 设置文本值渲染 1`] = `
<label class="root" tabindex="0">
  <span class="box" status="false"></span>
  <span vusion-slot-name="item"> </span>
</label>
`;
